### ISSUE TITLE
可以不通过cocoapods下载么

### ISSUE THREAD

**Issue by zhouzizi** [jgh-/VideoCore#325]
我电脑一直安装pod不成功。所以能提供正常的下载地址么

**Comment by notedit** [comment]
english please

**Comment by zhouzizi** [comment]
Could you give me a download link please? I can't install pod in my Mac..

**Comment by notedit** [comment]
https://github.com/jgh-/VideoCore/archive/master.zip

**Comment by zhouzizi** [comment]
No..I  want run the sample ,but my Mac don't has pod environment...

**Comment by yupuyang** [comment]
I want run the sample ,but my Mac don't have pod environment...

**Comment by ankur-sardar** [comment]
First Download pod by typing this in you terminal
$ sudo gem install cocoapods
